source/cloud/azure/azureml.md

```diff
@@ -173,7 +174,7 @@ ml_client.environments.create_or_update(env_docker_image)
 
 Now that we have our environment and custom logic, we can configure and run the `command` [class](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml?view=azure-python#azure-ai-ml-command) to submit training jobs.
 
-In a notebook cell, copy the example code from this documentation into a new folder.
+In a notebook cell, copy the example code from this documentation into a new folder, by running:
```

Comment:
Do we need the creation of the Dockerfile in the previous step, given this here?  If not, then Should the code below "Now create the Environment, making sure to label and provide a description:" go below the `%bash` script run?

Reply:
Looking at the order of the page the previous cell creates the `Environment` object, which uses the `Dockerfile`. So I don't think we need to create it again in this bash script.

Reply:
I see the bash script and some modifications to the Dockerfile in the section above where added by @jameslamb. James do you have any extra context about what seems like a duplication and what you've noticed while testing this? 

I personally like the idea of having the Dockerfile created by the bash script because it's easier, and in that case I'd move this part below the bash, right before the training. 
```python
env_docker_image = Environment(...)
```